backend/src/apps/owasp/models/snapshot_subscription.py

```
@@ -2,19 +2,36 @@
 
     class Meta:
         """Model options."""
 
         db_table = "owasp_snapshot_subscriptions"
         verbose_name_plural = "Snapshot Subscriptions"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["user", "name"],
+                name="unique_user_subscription_name",
+            ),
+        ]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)